tests/test_presets.py

```
@@ -501,7 +502,7 @@ def test_required_missing_config_raises(self, tmp_path: Path) -> None:
             ),
         )
 
-        with pytest.raises(SmolVMError, match="Required host config not found"):
+        with pytest.raises(SmolVMError, match="isn't there. Restore it"):
```

Ruff (0.15.15)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)